cabotage/scripts/create_admin.py

```diff
@@ -0,0 +1,27 @@
+from ..server import create_app, db
+from ..server.models import Organization, User
+from ..server.models.projects import Application, Project
+
+
+app = create_app()
```

Comment:
I think this command should short-circuit unless `DEBUG` is set to avoid letting it run outside of development

```
if not app.config["DEBUG"]:
    # somehow exit with a notification
```